launcher/ui/pages/modplatform/flame/FlamePage.cpp

```diff
@@ -166,7 +168,8 @@ void FlamePage::onSelectionChanged(QModelIndex first, QModelIndex second)
                 qWarning() << *response;
                 return;
             }
-            QJsonArray arr = doc.array();
+            // QJsonArray arr = doc.object().value("data").toArray();
```

Comment:
oopsie :P

Reply:
```
// QJsonArray arr = doc.object().value("data").toArray();
auto arr = Json::ensureArray(doc.object(), "data");
```

?

Reply:
remove the comment

Reply:
oh ok

forgot that there was git behind me if anything goes wrong :P